packages/rollup/src/generators/convert-to-inferred/convert-to-inferred.ts

```diff
@@ -0,0 +1,132 @@
+import {
+  formatFiles,
+  getProjects,
+  type Tree,
+  updateProjectConfiguration,
+} from '@nx/devkit';
+import { forEachExecutorOptions } from '@nx/devkit/src/generators/executor-options-utils';
+
+import { extractRollupConfigFromExecutorOptions } from './lib/extract-rollup-config-from-executor-options';
+import { RollupExecutorOptions } from '../../executors/rollup/schema';
+
+interface Schema {
+  project?: string;
+  all?: boolean;
```

Comment:
We can remove this, it should be the default behaviour of the generator. Rather than forcing a user to potentially pass it. We can update the conditional below to only check if project specifically is called.